static/js/hotkey.js

```diff
@@ -427,6 +451,10 @@ exports.process_hotkey = function (e, hotkey) {
     }
 
     if (hotkey.message_view_only && ui_state.home_tab_obscured()) {
+        // if up/down arrow key was pressed and streams menu is open
+        if ((event_name === 'up_arrow' || event_name === 'down_arrow') && subs.is_open) {
+            subs_arrow_keys(event_name);
+        }
```

Comment:
I think we should probably move these out of the `message_view_only` list and move this code into the `up/down` blocks a bit below.